lte/gateway/python/scripts/enodebd_cli.py

```diff
@@ -167,8 +167,14 @@ def get_enb_status(client, args):
     _print_str_status_line('GPS Longitude', enb_status.gps_longitude)
     _print_str_status_line('GPS Latitude', enb_status.gps_latitude)
     _print_str_status_line('FSM State', enb_status.fsm_state)
-
-
+    _print_str_status_line('GPS Aititude', enb_status.gps_aititude)
+    _print_str_status_line('Vendor', enb_status.vendor)
+    _print_str_status_line('MODEL Name', enb_status.model_name)
+    _print_prop_status_line('RF State', enb_status.rf_state)
```

Comment:
If this prints out as `RF State: true`, how do I interpret that? This should be made very obvious to somebody using the CLI for the first time.